lib/samples/filter_by_definition_expression_or_display_filter/filter_by_definition_expression_or_display_filter_sample.dart

```diff
@@ -82,24 +91,61 @@ class _FilterByDefinitionExpressionOrDisplayFilterSampleState
   }
 
   void applyDefinitionExpression() {
+    // remove the display filter
     _featureLayer.displayFilterDefinition = null;
+    // apply a definition expression to the feature layer
     _featureLayer.definitionExpression =
         "req_Type = 'Tree Maintenance or Damage'";
+    // count the number of features
+    calculateFeatureCount();
   }
 
   void applyDisplayFilter() {
+    // remove the definition expression
     _featureLayer.definitionExpression = '';
+    // apply a display filter to the feature layer
     final displayFilter = DisplayFilter.withWhereClause(
         name: 'Damaged Trees',
         whereClause: "req_type LIKE '%Tree Maintenance%'");
+    // create a manual display filter definition
     final manualDisplayFilterDefinition =
         ManualDisplayFilterDefinition.withFilters(
             activeFilter: displayFilter, availableFilters: [displayFilter]);
     _featureLayer.displayFilterDefinition = manualDisplayFilterDefinition;
+    // count the number of features
+    calculateFeatureCount();
   }
 
   void reset() {
+    // remove the definition expression and display filter
     _featureLayer.displayFilterDefinition = null;
     _featureLayer.definitionExpression = '';
+    // count the number of features
+    calculateFeatureCount();
+  }
+
+  void calculateFeatureCount() async {
+    // count the number of features
+    final queryParameters = QueryParameters();
+    // get the current extent of the map view
+    queryParameters.geometry = _mapViewController
+        .getCurrentViewpoint(viewpointType: ViewpointType.boundingGeometry)
+        ?.targetGeometry
+        .extent;
+    queryParameters.whereClause = '1=1';
```

Comment:
Is this line necessary? I think it should work without a `whereClause` as long as the other values are set properly on `_featureLayer`.

Reply:
Nope! it works without it just fine